src/pages/komodo-defi-framework/api/common_structures/index.mdx

```diff
@@ -334,6 +294,60 @@ The `numerator` and `denominator` are BigInteger numbers represented as a sign a
+| ------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| stream\_interval\_seconds | float   | Optional, defaults to `10` for balance events and `5` for network/heartbeat events. Interval in seconds between streaming event update requests.                                                                                                                                                                                                                                                                                                                                                                    |
+| estimate\_every           | float   | [stream::fee\_estimator::enable](/komodo-defi-framework/api/v20/streaming/fee_estimator/) only. Defaults to `15`. Interval in seconds between fee estimate updates.                                                                                                                                                                                                                                                                                                                                                 |
+| estimate\_type            | string  | [stream::fee\_estimator::enable](/komodo-defi-framework/api/v20/streaming/fee_estimator/) only. Defaults to `Simple`. If set to `Provider`, users must set the `gas_api` setting in their [MM2.json file](/komodo-defi-framework/setup/configure-mm2-json/) to source recommended fee values from third party providers [Infura](https://www.infura.io/) or [Blocknative](https://www.blocknative.com/) Used in [get\_eth\_estimated\_fee\_per\_gas](/komodo-defi-framework/api/v20/streaming/fee_estimator/) only. |
+| always\_send              | boolean | [stream::network::enable](/komodo-defi-framework/api/v20/streaming/network_enable/) only. Optional, defaults to `false`. If `true`, network data will always be sent every `stream_interval_seconds`, even when there is no change. If `false`, network data will be sent regardless.                                                                                                                                                                                                                               |
```

Comment:
these fields are present in two different configs but were merged into one.

`stream_interval_seconds` & `always_send` -> Network streamer config
`estimate_every` & `estimate_type` -> eth fee streamer config

Reply:
added in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/457/commits/bd009b74f6cd46b14487925155edb8a7e797c7da